src/sdk-reference/cpp/1/kuzzle/add-listener/snippets/add-listener.cpp

```diff
@@ -1,7 +1,7 @@
-EventListener listener = [](const std::string payload) {
+kuzzleio::EventListener listener = [](const std::string payload) {
   std::cout << payload << std::endl;
 };
-EventListener other_listener = [](const std::string payload) {
+kuzzleio::EventListener other_listener = [](const std::string payload) {
```

Comment:
`EventListener` is not in the `kuzzleio` namespace. There is a ticket with two tasks to change that: https://jira.kaliop.net/browse/KZL-500